google-cloud-bigtable/google-cloud-bigtable.gemspec

```diff
@@ -27,7 +28,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "minitest", "~> 5.10"
   gem.add_development_dependency "minitest-focus", "~> 1.1"
   gem.add_development_dependency "minitest-rg", "~> 5.2"
-  gem.add_development_dependency "rubocop", "~> 0.50.0"
+  gem.add_development_dependency "rubocop", "~> 0.56.0"
```

Comment:
This dependency version is incompatible with Ruby 2.0, and is why [CI failed on Ruby 2.0](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby/3860#tests/containers/5).